common/src/main/resources/ts/models/Response.ts

```diff
@@ -83,6 +97,16 @@ export class Responses extends Selection<Response> {
         }
     }
 
+    syncMineByQuestionIds = async (questionIds: number[], distributionId: number) : Promise<void> => {
+        try{
+            let data: IResponseResponse[] = await responseService.listMineByDistributionAndQuestionIds(questionIds, distributionId);
+            this.all = data.map((r:IResponseResponse) => new Response().build(r));
```

Comment:
En soi c'est ok, mais si tu peux mettre la partie typage dans le service direct l'avantage c'est que ton service pourra être modif pour renvoyer une Promise<Response[]> direct et recup ici pour simplement remplir ton this.all